homeassistant/components/ezviz/camera.py

```diff
@@ -181,6 +181,18 @@ def __init__(
         self._name = self.coordinator.data[self._idx]["name"]
         self._local_ip = self.coordinator.data[self._idx]["local_ip"]
 
+    @property
+    def device_state_attributes(self):
+        """Return the Ezviz-specific camera state attributes."""
+        return {
+            # Camera firmware version update available?
+            "upgrade_available": self.coordinator.data[self._idx]["upgrade_available"],
+            # camera's local ip on local network
+            "local_ip": self.coordinator.data[self._idx]["local_ip"],
+            # RTSP Stream
+            "RTSP stream": self._rtsp_stream,
+        }
+
```

Comment:
This is only way I can currently think of for a user to confirm the camera RTSP credentials. You can copy/paste the RTSP string in VLC Player (or other network steam viewer) as a way to test externally.

Reply:
When does the user need to confirm the RTSP credentials? Please explain the use case a bit more.

Reply:
This just makes troubleshooting exceptions easier. 

1) Advanced users trying to use different camera channels/streams will also be able to confirm that they configured options correctly etc. (Could also just get this from integration options) 

2) Confirm that you entered the correct password. - A user could also just re-add the integration. 

2) I could rather add the local IP as a sensor if that makes more sense? It's fetched from the EZVIZ api and someone could have a network condition preventing a camera from displaying a feed.

Reply:
1. Is not a string enough reason to add it to state attributes.
2.The password should be tested by the flow before it's saved in the config entry.
3. Yes we prefer individual sensors over state attributes.